api/admin/controller/collection_settings.py

```diff
@@ -28,6 +28,27 @@ def _get_collection_protocols(self):
         protocols = super(CollectionSettingsController, self)._get_collection_protocols(
             self.PROVIDER_APIS
         )
+
+        # dedupe and only keep the latest SETTINGS
+        # this will allow child objects to overwrite
+        # parent settings with the same key
+        # This relies on the fact that child settings
+        # are added after parent settings as such
+        # `SETTINGS + <configuration>.to_settings()`
```

Comment:
Is this indeed how things work, in general (i.e., elsewhere in the codebase)?

Reply:
Yep, https://github.com/ThePalaceProject/circulation/blob/main/core/opds2_import.py#L126